model-optimizer/extensions/ops/interpolate.py

```diff
@@ -69,3 +92,65 @@ def infer(node: Node):
         node.out_port(0).data.set_shape(output_shape)
 
         PermuteAttrs.create_permute_attrs(node, attrs=[('axes', 'input:0')])
+
+    @staticmethod
+    def get_axes(node: Node) -> np.ndarray:
+        opset = node.get_opset()
+        if opset in ['opset1', 'extension']:
```

Comment:
the operation may not have an opset `extension`.

Reply:
Fixed.